src/components/UI/AttachmentLink/AttachmentLink.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import styled from "@emotion/styled";
+import prettyBytes from "pretty-bytes";
+import { mixin, vars } from "../../../styles";
+import { SvgIcon, SvgIconPaperClip } from "./../../UI/SvgIcon";
+
+export interface AttachmentLinkProps {
+  className?: string;
+  filename: string;
+  data: string;
+  type: string;
+}
+
+export const AttachmentLinkUnStyled = ({ className, filename, data, type }: AttachmentLinkProps) => {
+  const decodedData = atob(data);
```

Comment:
![image](https://user-images.githubusercontent.com/7406870/85366919-dc8af600-b55a-11ea-974f-55c7d74feca9.png)

Are you seeing this on storybook?

Reply:
ok fixed